### PR TITLE
Move docs/services-firewalls.md into docs/user-guide/ and make small

### PR DESCRIPTION
**Pull request by davidopp**
wording changes to api.md

@bgrant0607 @thockin

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ What constitutes a compatible change and how to change the API are detailed by t
...
+To make it easier to eliminate fields or restructure resource representations, Kubernetes supports multiple API versions, each at a different API path prefix, such as `/api/v1beta3`. These are simply different interfaces to read and/or modify the same underlying resources. In general, all API resources are accessible via all API versions, though there may be some cases in the future where that is not true. 
 
-Distinct API versions present more clear, consistent views of system resources and behavior than intermingled, independently evolved resources. They also provide a more straightforward mechanism for controlling access to end-of-lifed and/or experimental APIs.
+We chose to version at the API level rather than at the resource or field level to ensure that the API presents a clear, consistent view of system resources and behavior, and to enable controlling access to end-of-lied and/or experimental APIs.
```
Comment (bgrant0607) [inline docs/api.md]: s/lied/lifed/

**Comment by bgrant0607** [comment]
Also update ./pkg/kubectl/cmd/create.go.

**Comment by davidopp** [comment]
Thanks for the review, PTAL. (And in the future I'll look in .go files for doc path changes-- didn't think of that.)

**Comment by bgrant0607** [comment]
Does running hack/run-gendocs.sh do anything? I'd expect the generated kubectl doc to change.

**Comment by bgrant0607** [comment]
Nevermind, I see that message wasn't in the help string.
LGTM

**Comment by bgrant0607** [comment]
cc @ArtfulCoder for merge

**Comment by davidopp** [comment]
squashed commits

**Comment by bgrant0607** [comment]
Merging

**Pull request merged**